doc/changes/changes_2.0.0.md

```diff
@@ -1,11 +1,12 @@
-# Integration-Test-Docker-Environment 1.7.0, released t.b.d.
+# Integration-Test-Docker-Environment 2.0.0, released t.b.d.
 
 ## Summary
 
+Version 2.0.0 of ITDE comes with breaking changes as public API class `DatabaseInfo` has been changed.  Former separate attributes `database_port_forward`and `bucketfs_port_forward` have been replaced by a single attribute `port` set to an instance of `PortForwarding` with attributes `database`, `bucketfs`, and `ssh`.
+
 Up to version 1.6.0 ITDE used `docker_exec` to access the Docker Container, e.g. to analyze the content of logfiles or execute some shell commands. With version 8 of Exasol database the format of the Docker Containers might change so that `docker_exec` is no longer possible. Instead ITDE will then need to use SSH access.
 
-The current release therefore enhances ITDE to enable to access the Docker Container via SSH.
-The user can select the docker access method with command line option `--db-os-access`, see User Guide.
+The current release therefore enhances ITDE to enable to access the Docker Container via SSH.  The user can select the docker access method with command line option `--db-os-access` and can specify a port number to which ITDE forwards the SSH port of the Docker Container, see User Guide.
```

Comment:
will go still into 1.7.0

Reply:
I will remove this part relying on to be mentioned in changes files for release 1.7.x.